### DISCUSSION TITLE
Roadrunner and lighthouse compatibility

### DISCUSSION THREAD

**Discussion by peihan-li** [bitcraze/discussions#666] [General]
Hi, I am wondering if I can use the lighthouse system with roadrunner. Currently, it seems the roadrunner only supports LPS on the website. Is there an on-going development for LH compatibility?

**Comment by tobbeanton** [comment]
No is the short and simple answer. Then it is better to use e.g. a bolt and add the LPS and lighthouse deck (which needs some modding to work together).
